Tests/UnitTests/Matchers/SizeTest.jsl

```diff
@@ -1,6 +1,30 @@
+﻿// Copyright © 2021, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
 
 ut test("SizeTests", "Avoids name collisions", Expr(
 	dt = New Table("A Table", Add Rows(1));
 	n rows = 5;
 	ut assert that(Expr(dt), ut n rows(1));
 ));
+
+ut test("SizeTests", "N Items", expr(
+	ut assert that(Expr({1,2,3}), ut n items(3))
+));
+
+ut test("SizeTests", "Length", expr(
+	ut assert value("ABC", ut length(3))
+));
+
+ut test("SizeTests", "Evaluation delayed until inner matcher", Expr(
+	ut assert that(Expr({1, 2, 3}), ut n rows(ut throws("NRow or NCol argument must be a Matrix or Data Table")));
```

Comment:
Update after fixing eval behavior
maybe `ut assert that(Expr(ut n rows({1, 2, 3}) << matches(3)), ut throws(...))`